java/clients/src/test/java/sleeper/clients/teardown/ShutdownSystemProcessesIT.java

```diff
@@ -100,6 +110,8 @@ void shouldShutDownCloudWatchRulesWhenSet() throws Exception {
                 .willReturn(aResponse().withStatus(200)));
         stubFor(listActiveClustersRequest()
                 .willReturn(aResponseWithNumRunningClusters(0)));
+        stubFor(listActiveApplicationsRequest()
+                .willReturn(aResponseWithNumRunningApplications(0)));
```

Comment:
It would be nice to extract the shared setup somewhere. Maybe a separate BeforeEach for each nested class, with a separate nested class for ECS tests and one for CloudWatch.

Reply:
I have done some general tidy up of the ShutdownSystemProcesseIT tests